src/db/ContentGenerator.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace src\db;
+
+use PDO;
+
+class ContentGenerator
+{
+    const LETTER_PATTERN = "/[a-zA-Z]/";
+
+    private array $sortNames = ['name', 'code', 'state', 'city'];
+
+    public function __construct(private Paginator $paginator)
+    {
+    }
+
+    public function request(array $data): array
+    {
+        $filters = $this->validateRequest($data);
+
+        return $this->getAirports($filters);
+    }
+
+    private function validateRequest(array $data): array
```

Comment:
In this method you return array of filters, so it's better to rename this method to `getFilters`